continuous-testing/starter-kits/README.md

```diff
@@ -1,3 +1,55 @@
 # Starter Kits
 
-Nothing here yet, come back soon... 
\ No newline at end of file
+This page contains starter kit information, which represent templates, code and configuration to help you get started quickly with continuous testing best practices described in this overall guide. Please see categories and links below for details. 
+
+## Software Composition Analysis
+
+This section contains links to sample actions, templates and configurations that analyze and validate composition of Open Source Software (OSS) components in software systems. Identifying software and licensing vulnerabilites and ensuring routine software updates is an [OSS cybersecurity](https://www.cisa.gov/uscert/ncas/alerts/aa22-137a) [best practice](https://appel.nasa.gov/2022/06/30/spotlight-on-lessons-learned-open-source-and-commercial-web-software-vulnerabilities/).  
+
+### Dependabot
+A GitHub ecosystem tool for dependency version and security vulnerability analysis.
+
+#### Automated Dependency Updates
+
+This Dependabot task provides an automated check for OSS component updates and automatically creates [pull requests](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates) to commit new versions. 
+
+Starter Kit:
+- [SLIM Starterkit Python -- Dependabot Script](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/develop/.github/dependabot.yml) to install in your GitHub repo
+
+To leverage this template, make sure to do the following:
+1. Discuss with your development team the cybersecurity best practice to regularly update OSS to latest versions and seek consensus on a workflow to accept proposed updates.
+2. Add Dependabot automation to your repository, either **manually** or via the **admin console** (choose **one**):
+> **Shortcut**  
+  Copy the pre-set configuration to an identical path in your repository, e.g. `.github/dependabot.yml`.
+   * **Admin console:** (requires admin rights)
+      1. Proceed to [enable Dependabot alerts through GitHub Settings UI](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/configuring-dependabot-alerts#enabling-or-disabling-dependabot-alerts-for-a-repository).
+   * **Manually:** (_approach available to all committers_)
+      1. Create an issue and an issue branch to implement a code change. Checkout the issue branch.
+      2. Copy the Dependabot configuration file from one of the SLIM Starterkit repos -- for example, [`dependabot.yml` in the Python Starterkit](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/dependabot.yml) -- into the root of your repository at `.github/dependabot.yml`.
```

Comment:
The link to the dependabot.yml currently points to a non-existent file on the main branch of the slim-starterkit-python repo. I think the dev branch just needs to be merged to main on the slim-starterkit-python repo.

Reply:
Yes, I think I did that purposely because the `main` branch is default and it's the only place it works, anyway. What I should do is to be sure to be consistent in where the links point (all the `main` branch, I think).

Reply:
Committed. ...